layouts/index.hbs

```diff
@@ -83,41 +83,22 @@
     justify-content: space-around;
   }
 
-  .interactive-item {
+  #interactive > * {
     border: 20px solid transparent;
     min-width: 1px;
   }
 
   @media (max-width: 480px) {
-    #logo-center {
-      display:none;
-    }
     #interactive {
       flex-wrap: wrap;
     }
   }
-
-  @media (min-width: 481px) {
-    #logo-first {
-      display:none;
-    }
-  }
   </style>
 
   <div id="interactive-wrapper">
     <div id="interactive">
-      <span id="logo-first" class="interactive-item">
-        <img src="/static/images/interactive/nodejs-interactive-logo-center.png" />
-      </span>
-      <a class="interactive-item" href="http://events.linuxfoundation.org/events/node-interactive-europe">
-        <img src="/static/images/interactive/nodejs-interactive-hero-banner-left-eu.png"
-          alt="Node.js Interactive Europe, Amsterdam, Netherlands, September 15-18, 2016"/>
-      </a>
-      <span id="logo-center" class="interactive-item">
-        <img src="/static/images/interactive/nodejs-interactive-logo-center.png"
-          alt="Node.js Interactive" />
-      </span>
-      <a class="interactive-item" href="http://events.linuxfoundation.org/events/node-interactive">
+      <img src="/static/images/interactive/nodejs-interactive-logo-center.png" />
```

Comment:
The wrapper span this is still required or the image will not be scaled well. This is how it looks in Chrome, responsive mode (Tablet - 768px):

![capture](https://cloud.githubusercontent.com/assets/1443911/18765005/1c4a2d12-8114-11e6-975a-56b4dcbcb289.PNG)